.github/workflows/helm.yml

```diff
@@ -116,17 +114,19 @@ jobs:
             exit 1
           fi
 
-
       - name: Check changelog
-        if: always()
+        if: ${{ ! inputs.publish-alpha }}
         run: |
-          VERSION_PATTERN="([0-9]+\.){2}[0-9]+(-alpha.[0-9][0-9]?)?"
+          VERSION_PATTERN="([0-9]+\.){2}[0-9]+"
```

Comment:
```suggestion
          VERSION_PATTERN="([0-9]+\.){2}[0-9]"
```

Reply:
Without the `+` we are limited to 1 digit for the patch segment.